tests/baselines/reference/abstractPropertyInConstructor.errors.txt

```diff
@@ -0,0 +1,25 @@
+tests/cases/compiler/abstractPropertyInConstructor.ts(4,24): error TS2715: Abstract property 'prop' in class 'AbstractClass' cannot be accessed in constructor.
+tests/cases/compiler/abstractPropertyInConstructor.ts(5,14): error TS2715: Abstract property 'prop' in class 'AbstractClass' cannot be accessed in constructor.
+
+
+==== tests/cases/compiler/abstractPropertyInConstructor.ts (2 errors) ====
+    abstract class AbstractClass {
+        constructor(str: string) {
+            this.method(parseInt(str));
+            let val = this.prop.toLowerCase();
+                           ~~~~
+!!! error TS2715: Abstract property 'prop' in class 'AbstractClass' cannot be accessed in constructor.
+            this.prop = "Hello World";
+                 ~~~~
+!!! error TS2715: Abstract property 'prop' in class 'AbstractClass' cannot be accessed in constructor.
+        }
+    
+        abstract prop: string;

```

Comment:
can you add a callback case too, something like `abstract cb: (s: string) => void;`?